### ISSUE TITLE
Make Debug.Control.choice take a nonempty list

### ISSUE THREAD

**Issue by MartinSStewart** [avh4/elm-debug-controls#11]
In the documentation for [Debug.Control.choice](https://package.elm-lang.org/packages/avh4/elm-debug-controls/latest/Debug-Control#choice) it says 
> This will crash if you provide an empty list.

Why not change the type signature to `( String, Control a ) -> List ( String, Control a ) -> Control a` so that can never happen?

**Comment by avh4** [comment]
Yeah, I think that's a good change.  It wasn't done originally because when I ported this to Elm 0.19, it wasn't yet clear what the community preference would be w/r to this sort of API, but I think your proposed type makes sense now.
